pre_commit/clientlib.py

```diff
@@ -156,6 +156,30 @@ def check(self, dct: Dict[str, Any]) -> None:
             )
 
 
+class OptionalSensibleSlashesAtHook(cfgv.OptionalNoDefault):
+    def check(self, dct: Dict[str, Any]) -> None:
+        super().check(dct)
+
+        if r'[\/]' in dct.get(self.key, ''):
+            logger.warning(
+                f'Pre-commit normalizes the slashes in {self.key!r} field in '
+                f"hook {dct.get('id')!r} to forward slashes, so you can use "
+                f'/ instead of [\\/]',
+            )
+
+
+class OptionalSensibleSlashesAtTop(cfgv.OptionalNoDefault):
+    def check(self, dct: Dict[str, Any]) -> None:
+        super().check(dct)
+
+        if r'[\/]' in dct.get(self.key, ''):
+            logger.warning(
+                f'Pre-commit normalizes the slashes in the top-level '
+                f'{self.key!r} field to forward slashes, so you can use / '
+                f'instead of [\\/]',
```

Comment:
you can use `fr'...'` for a raw f-string by the way

Reply:
Fixed.